js/plot/util/scales.js

```diff
@@ -21,7 +21,9 @@ var _ = require('lodash');
 var commonbolus = require('./commonbolus');
 
 var scales = function(opts) {
-  opts = opts || {};
+  opts = _.assign({}, opts) || {};
+
+  var GLUCOSE_MM = 18.01559;
```

Comment:
Just factored it out into a `js/data/util/constants` as I know you're right. Added a test to make sure it stays at 18.01559 or we'll learn about it, too.